src/js/utils/dom-utils.js

```diff
@@ -71,7 +76,11 @@ function addClassName(element, className) {
 }
 
 function normalizeTagName(tagName) {
-  return tagName.toLowerCase();
+  if (!tagName) {
```

Comment:
And this can probably be reverted if you move the fix as suggested in my comment. IMO this function should not fail silently as you changed it to: If there is no `tagName` to be normalized our code is definitely not doing what we expected it to do.